pennylane/templates/subroutines/out_adder.py

```diff
@@ -27,25 +27,28 @@ class OutAdder(Operation):
 
     .. math::
 
-        \text{OutAdder}(mod) |x \rangle | y \rangle | b \rangle = |x \rangle | y \rangle | b+x+y \, \text{mod} \, mod \rangle,
+        \text{OutAdder}(mod) |x \rangle | y \rangle | b \rangle = |x \rangle | y \rangle | b+x+y \; \text{mod} \; mod \rangle,
 
     The implementation is based on the quantum Fourier transform method presented in
     `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
 
     .. note::
 
-        Note that :math:`x` and :math:`y` must be smaller than :math:`mod` to get the correct result.
+        To obtain the correct result, :math:`x`, :math:`y` and :math:`b` must be smaller than :math:`mod`.
+
+    .. seealso:: :class:`~.PhaseAdder` and :class:`~.Adder`.
 
     Args:
         x_wires (Sequence[int]): the wires that store the integer :math:`x`
         y_wires (Sequence[int]): the wires that store the integer :math:`y`
-        output_wires (Sequence[int]): the wires that store the addition result
-        mod (int): the modulus for performing the addition, default value is :math:`2^{\text{len(output\_wires)}}`
-        work_wires (Sequence[int]): the auxiliary wires to use for the addition
+        output_wires (Sequence[int]): the wires that store the addition result. If the register is in a non-zero state :math:`b`, the solution will be added to this value.
+        mod (int): the modulo for performing the addition. If not provided, it will be set to its maximum value, :math:`2^{\text{len(output_wires)}}`.
+        work_wires (Sequence[int]): the auxiliary wires to use for the addition. The work wires are not needed if :math:`mod=2^{\text{len(x_wires)}}`, otherwise two work wires should be provided. Defaults to ``None``.
```

Comment:
```suggestion
        work_wires (Sequence[int]): the auxiliary wires to use for the addition. The work wires are not needed if :math:`mod=2^{\text{len(output_wires)}}`, otherwise two work wires should be provided. Defaults to ``None``.
```